@here/harp-mapview/lib/MapView.ts

```diff
@@ -671,6 +679,11 @@ export class MapView extends THREE.EventDispatcher {
 
     private m_skyBackground?: SkyBackground;
     private m_createdLights?: THREE.Light[];
+    private m_directionalLight?: THREE.DirectionalLight;
+
+    get directionalLight(): THREE.DirectionalLight | undefined {
```

Comment:
Why? If we really have to expose the lights in `MapView` we should expose them generic (i.e. `THREE.Light[]`).
Is this just for convenience, so that the user does not need to go through the scene?
Honestly right now I would not add this API at all and just search for the lights in the scene.
Eventually the lights should be updateable via. `MapView.setDynamicProperty` but right now this only works for `Technique` attributes. Nevertheless we should not add an API right now that we will deprecate later.